CONVENTIONS.md

```diff
@@ -29,6 +31,129 @@ Black is left at defaults, flake8 and isort are configured to adhere to black.
 
 All functions need to be covered by unit tests.
 
+# Coordinate frame conventions
+
+The primary purpose of pylinalg as a library is to support linear algebra
+operations in pygfx and applications based on pygfx. As such, pylinalg shares
+its coordinate frame conventions with pygfx, which in turn follows [gITF's
+conventions
+](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#coordinate-system-and-units).
+
+## Coordinate Systems
+
+Unless stated otherwise, frames use 3-dimensional euclidean coordinates and a
+right-handed coordinate frame. What differs depending on the type are the
+semantics used to describe each axis. Distances are measured in `m` (meters) and
+and angles in `rad` (radians) unless stated otherwise.
+
+* **Object Coordinates** are represented using `(x, y, z)` vectors and are used
+  to describe objects in a scene. They use the following semantics:
+  * The *negative* X axis is called *right*.
+  * The positive Y axis is called *up*.
+  * The positive Z axis is called *forward*.
+* **Camera Coordinates** are represented using `(x, y, z)` vectors and are used
+  to describe cameras and lights. They use the following semantics:
+  * The positive X axis is called *right*.
+  * The positive Y axis is called *up*.
+  * The *negative* Z axis is called *forward*.
+* **Normalized Device Coordinates (NDC)** are left-handed and represented using
+  `(x, y, z)` vectors. They are used to describe points to render/plot. Points
+  inside the unit (half) box are rendered, others are not. NDC uses the
+  following semantics:
+  * The positive X axis is called *right* and the box extent is `[-1, 1]`.
+  * The positive Y axis is called *up* and the box extent is `[-1, 1]`.
+  * The positive Z axis is the viewing direction and the box extent is `[0, 1]`.
+* **Spherical Coordinates** are represented using `(radius, theta, phi)` vectors
+  and use the following semantics:
+  * The *radius* measures the distance between a point and the origin and lies
+    between `[0, inf)`.
+  * The *phi* angle measures the counter-clockwise (CCW) rotation around the
+    positive y-axis. It is measured from the positive Z-axis and lies between
+    `[0, pi)`.
+  * The *theta* angle measures the counter-clockwise (CCW) rotation around the
+    negative X-axis. It is measured from the positive Y-axis and lies between
+    `[0, 2*pi)`.
```

Comment:
I've updated the spherical coordinates to reflect the definition that the current implementation uses.